src/confcom/azext_confcom/README.md

```diff
@@ -143,123 +171,158 @@ Users just need to make a tar file by using the `docker save` command above, inc
+```
+
+## dmverity Layer Hashing
+
+To ensure the container that is being deployed is the intended container, the `confcom` tooling uses [dmverity hashing](https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html). This is done by downloading the container locally with the Docker Daemon (or using a pre-downloaded tar file of the OCI image) and performing the dmverity hashing using the [dmverity-vhd tool](https://github.com/microsoft/hcsshim/tree/main/cmd/dmverity-vhd). These layer hashes are placed into the Rego security policy in each container's "layers" field. Note that these dmverity layer hashes are different than the layer hashes reported by `docker image inspect`.
+
+### Mixed-mode Policy Generation
+
+An OCI image can be made available for policy generation in three ways:
+
+1. The image is in the local Docker Daemon and can be found either with its image and tag names or sha256 hash.
+2. The image is in an accessible remote repository. Usually this is either Docker Hub or Azure Container Registry. Note that if the registry is private, you must log in prior to policy generation.
+3. The image is locally saved as a tar file in the form specified by `docker save`.
+
+Mixed-mode policy generation is available in the `confcom` tooling, meaning images within the same security policy can be in any of these three locations with no issues.
+
+## Security Policy Information Sources
+
+Each container in a security policy can get its information from two different sources:
+
+1. The image manifest. This can be explored using `docker image inspect`
+2. The ARM Template used to generate the security policy. This can be used for startup command, environment variables, etc.
+
+The `confcom` tooling uses the image manifest for default values and then adds or overwrites those values using what is found in the ARM Template. The API Reference for defining values in the [ARM Template can be found here](https://learn.microsoft.com/en-us/azure/templates/microsoft.containerinstance/containergroups?pivots=deployment-language-arm-template)
 
 ## Security Policy Rules Documentation
 
 Below is an example rego policy:
 
-    ```rego
-    package policy
+```rego
+package policy
 
-    import future.keywords.every
-    import future.keywords.in
+import future.keywords.every
+import future.keywords.in
 
-    api_version := "0.10.0"
-    framework_version := "0.1.0"
+api_version := "0.10.0"
+framework_version := "0.1.0"
 
-    fragments := [...]
+fragments := [...]
 
-    containers := [...]
+containers := [...]
 
-    allow_properties_access := false
-    allow_dump_stacks := false
-    allow_runtime_logging := false
-    allow_environment_variable_dropping := true
-    allow_unencrypted_scratch := false
+allow_properties_access := false
+allow_dump_stacks := false
+allow_runtime_logging := false
+allow_environment_variable_dropping := true
+allow_unencrypted_scratch := false
```

Comment:
Should add: https://msr-gryffindor.visualstudio.com/Taro/_wiki/wikis/GCS%20Policy%20Breakdown/47/framework?anchor=%60allow_capabilities_dropping%60#:~:text=allow_capabilities_dropping